dashboard/app/views/levels/editors/_netsim.html.haml

```diff
@@ -22,37 +26,102 @@
       %td= f.label :show_add_router_button, 'Show "Add Router/Room" Button'
-      %td Whether the "Add Router" (or "Add Room") button should be available to students.
+      %td
+        Whether the "Add Router" (or "Add Room") button should be available
+        to students.
 
 %h3 Simulation Configuration
 .field
   = f.label :message_granularity, 'Message Granularity'
-  %p Whether the simulator puts a single bit into storage at a time, or a whole packet. Should use 'bits' for variant 1 (levels about the coordination problem), and 'packets' for levels where the coordination problem is abstracted away.
+  %p
+    Whether the simulator puts a single bit into storage at a time, or a whole
+    packet. Should use 'bits' for variant 1 (levels about the coordination
+    problem), and 'packets' for levels where the coordination problem is
+    abstracted away.
   = f.select :message_granularity, options_for_select(@level.class.message_granularity_options, @level.message_granularity)
+  %p
+    Note: Using "packet" granularity just means a whole message is sent when
+    you click Send.  It does not necessarily:
+    %ul
+      %li Surface the concept of a "packet" to the student
+      %li Limit packet size (see "Default packet size")
+      %li Let the student compose multiple packets (see "Show 'Add Packet' button")
+      %li Use headers or packet counts (see "Client packet header format")
 .field
   %table.checkboxes
     %tr
       %td= boolean_check_box f, :automatic_receive
       %td(nowrap)= f.label :automatic_receive, 'Auto-receive'
-      %td Whether the local node will watch the message table and automatically pick up messages directed to itself, dumping them into the message log.  If false, some other method must be used for receiving data (like a manual 'receive' button)
+      %td
+        Whether the local node will watch the message table and automatically
+        pick up messages directed to itself, dumping them into the message log.
+        If false, some other method must be used for receiving data (like a
+        manual 'receive' button)
     %tr
       %td= boolean_check_box f, :broadcast_mode
       %td(nowrap)= f.label :broadcast_mode, 'Broadcast Mode'
-      %td Turns routers into "rooms" where any packets sent will go to everyone present in the room. (Recommended to hide the Router tab when using this)
+      %td
+        Turns routers into "rooms" where any packets sent will go to everyone
+        present in the room. (Recommended to hide the Router tab when using this)
 
 %h3 Packet Headers
 .field
-  %p Packet headers can be customized to different header column combinations, orderings, and sizes with a JSON spec.  This can safely be left empty <tt>[]</tt> in "Broadcast Mode."
-  %p Example: <tt>[{"key":"toAddress", "bits":8}, {"key":"fromAddress", "bits":8}]</tt>
-  %p Valid header keys are "toAddress", "fromAddress", "packetIndex" and "packetCount"
+  = f.label :address_format, 'Address format'
```

Comment:
Here are the two new fields, and changes to the documentation around the header spec fields.